.github/workflows/image.yaml

```diff
@@ -56,8 +45,10 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Build image
         env:
-          IMAGE_NAME: ghcr.io/${LOWERCASE_REPO_OWNER}/k8s-dra-driver
-          VERSION: ${COMMIT_SHORT_SHA}
+          IMAGE_NAME: ghcr.io/nvidia/k8s-device-plugin
```

Comment:
This doesn't seem right

Reply:
My bad, I was testing the workflows and forgot to edit this bit, it is now fixed